tests/mkdocs_i18n_fixed_link.yml

```diff
@@ -0,0 +1,13 @@
+site_name: MkDocs static i18n plugin tests with fixed link
```

Comment:
This file is never used AFAIK, drop it please

Reply:
Sorry I forgot to change the reference in the test. It should be solved with [f5890d98ab70123b07fefca717b478c35f760a9f](https://github.com/dariocurr/mkdocs-static-i18n/commit/f5890d98ab70123b07fefca717b478c35f760a9f)